cypress/integration/tableAjax.feature

```diff
@@ -0,0 +1,58 @@
+Feature: Table Ajax component
+  I want to change Table component properties
```

Comment:
Feature is for Table Ajax,  but U want to change just Table :)

Reply:
fixed :)